### DISCUSSION TITLE
[Login Site] Is it horizontally scalable?

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#434] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v12.0.2

### Description

Hi guys,
I have a doubt about the Login Site. I know it's used to complete the login flow and to keep information about that flow (code, state and so on).

Is it scalable, right? Or it caches some data that can break the login flow if we scale its replicas?

Thanks.

**Comment by davidebianchi** [comment, marked as answer]
Hi @silversoul93! It is scalable, it only gives some static files
